### ISSUE TITLE
‘Border’ popup controls as ‘Line width', 'Line style’ and all colors are not accessible using keyboard.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#83]
User Experience:

User rely on keyboard/Screen Reader will find difficulty to interact with the controls if controls are not accessible through keyboard as user won't be able to perform functionality over such control.

 

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode

4. Navigate to the Left nav and select "Styles" button.

5. Navigate to style page and select Heading Level 1.

6. Navigate to the various controls of the "Heading Level 1" dialog box.

7. Navigate to box toggle button and activate it.

8. Navigate to border controls and activate it.

9. Verify whether opened ‘Border’ popup controls are accessible or not via keyboard.

 

Actual Result:

‘Border’ popup controls as ‘Line width' input field,  'Line style’ combobox and all colors are not accessible using keyboard.

 
Expected Result:

‘Border’ popup controls as ‘Line width' input field,  'Line style’ combobox   and all colors should be accessible using keyboard.

 

MAS Reference:

MAS 2.1.1 - Keyboard (36)

Note:

Similar issue exist with Chrome.

Similar issue exist with all border controls.

Similar issue exist throughout the application.

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.
